app-text/nfoview/nfoview-2.1.ebuild

```
@@ -0,0 +1,78 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit desktop python-single-r1 xdg virtualx
```

```suggestion
inherit desktop python-single-r1 virtualx xdg
```

Keep it sorted, unless you have a very good reason not to.

---

fixed